audmath/core/api.py

```diff
@@ -677,6 +677,79 @@ def samples(
     return round(duration * sampling_rate)
 
 
+def similarity(
```

Comment:
Are we planning to add additional similarities in the further? If so it might be useful to use a more meaningful name like cosine_similarity. Or we could add an "metric" parameter with "cosine" as default not implemented for other similarity metrics.

Reply:
Okay, perfect. I see that you already put more thought into this than I did.